src/BTVNanoCommissioning/workflows/ctag_eWc_valid_sf.py

```diff
@@ -122,6 +122,9 @@ def process_shift(self, events, shift_name):
             (events.Electron.pt > 34) & ele_mvatightid(events, self._campaign)
         ]
         req_ele = ak.count(iso_ele.pt, axis=1) == 1
+        jet_sel = jet_id(events, self._campaign) & (
+            ak.all(events.Jet.metric_table(iso_ele) > 0.5, axis=2)
+        )
```

Comment:
I think the additional `ak.fill_none` for the selection to avoid `None` in the jet selection. (just to be sure None filtered out)